cypress/support/config/services.js

```diff
@@ -2308,7 +2330,10 @@ const genServices = {
         paths: {
           live: [],
           test: [],
-          local: ['/turkce/haberler-dunya-50924340'],
+          local: [
```

Comment:
Should this be in the test section instead of local?

Reply:
This array is within the PGPs object - should we be changing it at all?

Reply:
Fixed